skimage/metrics/tests/test_simple_metrics.py

```diff
@@ -95,3 +95,23 @@ def test_NRMSE_errors():
     # invalid normalization name
     with testing.raises(ValueError):
         normalized_root_mse(x, x, normalization='foo')
+
+
+def test_nmi():
+    assert_almost_equal(normalized_mutual_information(cam, cam), 2)
+    assert (normalized_mutual_information(cam, cam_noisy)
+            < normalized_mutual_information(cam, cam))
+
```

Comment:
Could you please add a test with a 3d image and different sizes to be sure that it works also in 3d? (It should from what I read).